src/App.js

```diff
@@ -1,65 +1,58 @@
-import './App.css';
-import { BrowserRouter as Router, Route,Switch,Redirect } from "react-router-dom";
-import SignUpComponent from './components/SignUpComponent';
-import Home from './pages/NotFound';
-import SignInComponent from './components/SignInComponent';
-import Dashboard from './pages/Dashboard';
-import Students from './pages/Students';
-import { useState } from 'react';
-import { useEffect } from 'react';
-import NotFound from './pages/NotFound';
-import Coaches from './pages/Coaches';
-import Studentinfo from './pages/StudentInfo';
-import Coachinfo from './pages/CoachInfo';
-import Attendance from './pages/Attendance';
-import Reports from './pages/Reports';
-import Reportbyname from './pages/ReportByName';
-import Reportbydaterange from './pages/ReportByDateRange';
-
-
+import "./App.css";
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+  Redirect,
+} from "react-router-dom";
+import Home from "./pages/NotFound";
+import Dashboard from "./pages/Dashboard";
+import Students from "./pages/Students";
+import NotFound from "./pages/NotFound";
+import Coaches from "./pages/Coaches";
+import Studentinfo from "./pages/StudentInfo";
+import Coachinfo from "./pages/CoachInfo";
+import Attendance from "./pages/Attendance";
+import Reports from "./pages/Reports";
+import Reportbyname from "./pages/ReportByName";
+import Reportbydaterange from "./pages/ReportByDateRange";
+import SignInComponent from "./pages/SignInComponent";
+import SignUpComponent from "./pages/SignUpComponent";
 
 function App() {
-
-
   return (
     <div>
-
       <Router>
         <Switch>
-
-          {< Route path="/dashboard">
+          <Route path="/dashboard">
             <Dashboard />
-          </Route>}
-
-
-          {<Route path="/students" exact>
+          </Route>
+          <Route path="/students" exact>
```

Comment:
You can use prettier for formatting. You can install the prettier plugin in vscode. Also enable format on save in your vscode settings. That way your files will be auto-formatted when ever you save.

Reply:
formatted all files